app.js

```diff
@@ -2,6 +2,9 @@ const express = require("express");
 const app = express();
 const port = 3000;
 
+app.use(express.static("public"));
+app.use("/", express.static("public/index.html"));
```

Comment:
Do we need to specify that it serve this particular file? What does `app.use(express.static("public"))` do for us already?